PusherClient/PusherOptions.cs

```diff
@@ -24,9 +24,9 @@ public PusherOptions()
         public bool Encrypted { get; set; }
 
         /// <summary>
-        /// Gets or set the <see cref="IAuthorizer"/> to use.
+        /// Gets or set the <see cref="IChannelAuthorizer"/> to use.
         /// </summary>
-        public IAuthorizer Authorizer { get; set; } = null;
```

Comment:
Should we keep both for backwards compatibility?

Reply:
I guess so. Maybe adding some Deprecated comment or something

Reply:
done